app/forms/webauthn_setup_form.rb

```diff
@@ -46,13 +47,13 @@ def platform_authenticator?
 
   attr_reader :success, :transports, :invalid_transports
   attr_accessor :user, :challenge, :attestation_object, :client_data_json,
-                :name, :platform_authenticator, :authenticator_data_flags
+                :name, :platform_authenticator, :authenticator_data_flags, :nice_name
 
   def consume_parameters(params)
     @attestation_object = params[:attestation_object]
     @client_data_json = params[:client_data_json]
-    @name = params[:name]
     @platform_authenticator = (params[:platform_authenticator].to_s == 'true')
+    @name = @platform_authenticator ? @device_name : params[:name]
```

Comment:
An idea: Maybe rather than targeting platform authenticator's specifically, we can allow the name to be provided either through the constructor or through the `submit` method, and then assign it directly to the instance's `name`.

I'd also be curious your thoughts on [an earlier discussion in Slack](https://gsa-tts.slack.com/archives/C0NGESUN5/p1701183890866009) about whether it makes sense for `name` to be provided through the constructor or through the `submit` method. I could see arguments either way.